gnap/api.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package gnap
```

Comment:
the project structure seems weird - gnap module is sibling to pkg and cmd?
@rolsonquadras @Baha-sk @fqutishat

Reply:
should it be moved to `pkg`? it would be weird to have a submodule under `pkg` though

Reply:
@troyronda - are you suggesting we move this under `pkg` or have some other folder at top-level like `component` as is in AFG https://github.com/hyperledger/aries-framework-go/tree/main/component.